.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+.github/**/*    @ed-fi-exchange-oss/blackbird-automation-owners
```

Comment:
This is a new security feature we're trying out across Ed-Fi projects. Changes to `.github/` files must be approved by a member of the `blackbird-automation-owners` team. On all Ed-Fi scrum teams we'll give permission to the product owner, me, and one person from the dev team (Brad, in this case), so that one of the three of us can review and approve the change.

Of course in this case I can't approve my own change... so even if someone else does most of the review for this ticket, DavidC or Brad will also need to approve.

@bradbanister @david-clements-ed-fi @stevenarnold-ed-fi @jleiva-gap @DavidJGapCR